specs/latest/1.0/index.html

```diff
@@ -3565,10 +3565,7 @@ <h2 class="no-toc">Table of contents</h2>
 <h3><a name="READS_OUTSIDE_FRAMEBUFFER">Reading Pixels Outside the Framebuffer</a></h3>
 
 <p>
-In the WebGL API, functions which read the framebuffer
-(<code>copyTexImage2D</code>, <code>copyTexSubImage2D</code>, and <code>readPixels</code>) are
-defined to generate the RGBA value (0, 0, 0, 0) for any pixel which is outside of the bound
-framebuffer.
+In the WebGL API, three functions read the framebuffer: <code>copyTexImage2D</code>, <code>copyTexSubImage2D</code>, and <code>readPixels</code>. <code>copyTexImage2D</code> is defined to generate the RGBA value (0, 0, 0, 0) for any pixel outside of the bound framebuffer. <code>copyTexSubImage2D</code> and <code>readPixels</code> are defined not to touch the corresponding destination range for any pixel outside the bound framebuffer.
```

Comment:
@zhenyao Please keep the wrap the text to no more than 100 columns.  It's painful to scroll these long sentences.
I would also like to see a corresponding copyTexSubImage2D test that reflects the new behavior.